build.rs

```diff
@@ -0,0 +1,9 @@
+extern crate prost_build;
+
+fn main() {
+    prost_build::compile_protos(
+        &["src/wasmv1_execution/abi/proto/abi.proto"],
```

Comment:
we should ensure that this works on mac and windows

Reply:
perfect !

Reply:
Can you also make an update on massa-docs ? Especially this page: https://docs.massa.net/en/latest/testnet/install.html#from-source-code-advanced-installation

Reply:
Please see https://github.com/massalabs/massa-docs/pull/148

I kept doc minimal to avoid duplicating it. 

I suggest as a followup that we move what's related to protobuf from the massa-grpc README.md (mostly copied here) into massa-docs then that we make both README.md to refer to massa-docs.

Reply:
sounds good